### ISSUE TITLE
Swoole server not load ziggy-js

### ISSUE THREAD

**Issue by folkevil** [hhxsv5/laravel-s#288]
Hi,
My project is using [tightenco/ziggy](https://github.com/tightenco/ziggy) to generate in js. I  tried to include ziggy-js to head section, or import in app.js file (install via npm). Everything is works fine if I run with php artisan serve, but if I run laravels starts, the website loaded all js files (included ziggy-js file), but it still shows error "route is not defined // Ziggy is not defined "

Do you have any suggestions?

**Comment by hhxsv5** [comment]
Did you forget to enable the config [handle_static](https://github.com/hhxsv5/laravel-s/blob/master/Settings.md#handle_static)?

**Comment by fynntang** [comment]
@hhxsv5 大佬我遇到了和这个相同的问题，
我的`handle_static` 是配置为true的，我重启后第一次是可以正常访问的，但是一刷新就报 `Ziggy is not defined`

![QQImage20201129222720.png](https://i.loli.net/2020/11/29/pBbvZoKwYCPAUOQ.png)

**Comment by folkevil** [comment]
Sorry for late reply, I got same problem as @fynntang mentioned before

**Comment by fynntang** [comment]
@folkevil Did you use `inertiajs/inertia-laravel` bag,I haven't solved it yet

**Comment by folkevil** [comment]
@fynntang I have 2 projects, 1 with inertiajs but got the same problem. Still don't know why.

**Comment by folkevil** [comment]
@fynntang @hhxsv5 I tried this way:
in blade.php:

 ```
<script src="https://unpkg.com/ziggy-js@0.9.x/dist/js/route.min.js"></script>
    <script>
        var Ziggy = {
            namedRoutes: {},
            baseUrl: '{!! env('APP_URL','http://127.0.0.1:5200') !!}',
            baseProtocol: '{!! env('LARAVELS_LISTEN_PROTOCOL','http') !!}',
            baseDomain: '{!! env('LARAVELS_LISTEN_IP', '127.0.0.1') !!}',
            basePort: '{!! env('LARAVELS_LISTEN_PORT', '5200') !!}',
            defaultParameters: [],
        };

        if (typeof window !== 'undefined' && typeof window.Ziggy !== 'undefined') {
            for (var name in window.Ziggy.namedRoutes) {
                Ziggy.namedRoutes[name] = window.Ziggy.namedRoutes[name];
            }
        }

        var routes = {!! config('ziggy.routes') !!};

        for (var name in routes) {
            Ziggy.namedRoutes[name] = routes[name];
        }
    </script>
```

and in ziggy.php I added an array generated routes of Laravel when ran without swoole server and it worked. 

I don't know why, but I guess the routes Laravel generated when run with Swoole server are different with routes with normal php server.

**Comment by hhxsv5** [comment]
Need to reset `BladeRouteGenerator::$generated`.

Try to add ZiggyCleaner.

```php
<?php
namespace App\Cleaners;
use Hhxsv5\LaravelS\Illuminate\Cleaners\BaseCleaner;
use Tightenco\Ziggy\BladeRouteGenerator;
class ZiggyCleaner extends BaseCleaner
{
    public function clean()
    {
        BladeRouteGenerator::$generated = false;
    }
}
```

**Comment by hhxsv5** [comment]
If the problem is solved, LaravelS v3.7.21 will be released, which will have built-in ZiggyCleaner
